package.json

```diff
@@ -12,31 +15,20 @@
   "bugs": {
     "url": "https://github.com/seek-oss/tsconfig-seek/issues"
   },
-  "scripts": {
-    "commit": "git-cz",
-    "semantic-release": "semantic-release"
-  },
-  "husky": {
-    "hooks": {
-      "commit-msg": "commitlint --edit --extends seek"
-    }
-  },
   "homepage": "https://github.com/seek-oss/tsconfig-seek#readme",
   "devDependencies": {
-    "@commitlint/cli": "^7.2.1",
-    "commitizen": "^3.0.2",
-    "commitlint-config-seek": "^1.0.0",
-    "cz-conventional-changelog": "^2.1.0",
-    "husky": "^1.1.2",
-    "semantic-release": "^15.9.17"
+    "@changesets/cli": "^2.26.0",
+    "@changesets/get-github-info": "^0.5.2",
+    "typescript": "^5.0.4"
   },
-  "release": {
-    "success": false
-  },
-  "config": {
-    "commitizen": {
-      "path": "./node_modules/cz-conventional-changelog"
-    }
+  "scripts": {
+    "release": "changeset publish",
+    "test": "tsc --noEmit",
+    "changeset-version": "changeset version && prettier --write ."
   },
-  "tsconfig": "tsconfig.json"
+  "tsconfig": "tsconfig.json",
```

Comment:
Not sure what this is required for?

Reply:
I can't readily find any documentation about this property. I reckon we just remove it.

```suggestion
```